src/tests/ascent/t_ascent_relay.cpp

```diff
@@ -988,6 +988,77 @@ TEST(ascent_relay, test_relay_sparse_topos)
 }
 
 
+
+//-----------------------------------------------------------------------------
+TEST(ascent_relay, test_relay_lor_extract)
+{
+    Node n;
+    ascent::about(n);
+    //
+    // Create an example mesh.
+    //
+    Node data,verify_info;
+    conduit::relay::io::blueprint::read_mesh(test_data_file("taylor_green.cycle_001860.root"),
+                                             data);
+    EXPECT_TRUE(conduit::blueprint::mesh::verify(data,verify_info));
+
+    string output_path = prepare_output_dir();
+    string output_file = conduit::utils::join_file_path(output_path,"tout_relay_ho_no_lor");
+
+    conduit::Node actions;
+    // add the extracts
+    conduit::Node &add_extracts = actions.append();
+    add_extracts["action"] = "add_extracts";
+    Node &extracts = add_extracts["extracts"];
+
+    extracts["e1/type"]  = "relay";
+
+    extracts["e1/params/path"] = output_file;
+    extracts["e1/params/protocol"] = "blueprint";
+
+    std::cout << actions.to_yaml() << std::endl;
+
+    //
+    // Run Ascent
+    //
+    Ascent ascent;
+    ascent.open();
+    ascent.publish(data);
+    ascent.execute(actions);
+    ascent.close();
```

Comment:
Do we want a EXPECT_TRUE file check for the first extract?

Reply:
thanks, done!